### ISSUE TITLE
Move the existing forward simulations setup to Titan

### ISSUE THREAD

**Issue by mturilli** [radical-collaboration/hpc-workflows#7]

**Comment by vivek-bala** [comment]
Current issue: https://github.com/radical-cybertools/radical.pilot/issues/1310

**Comment by mturilli** [comment]
Issue radical-cybertools/radical.pilot#1310 is closed. Is it still blocking this action?

**Comment by vivek-bala** [comment]
No, that issue isn't blocking anymore. The RP script works on Titan. The corresponding entk script (that we have for stampede + was tried out by Matthieu, Wenjie) is not ready since entk has to be brought up to the changes in RP devel.
